test-reporting/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd prefer if we could do this without an additional module.
Maybe this could be done in the phoenix-assembly module ?